pkg/apis/kubevirt/types_cloudprofile.go

```diff
@@ -45,8 +48,18 @@ type MachineImageVersion struct {
 	SourceURL string
 }
 
-// MachineDeploymentConfig defines deployment config for VMs for specific machine type
-type MachineDeploymentConfig struct {
-	// MachineTypeName is the name of the machine type, used as a reference to MachineType object
-	MachineTypeName string
+// MachineTypes extend machineType object to KubeVirt provider specific config
+type MachineTypes struct {
```

Comment:
The name of this type should be `MachineType`, as it refers to a single machine type. `MachineImages` is similarly wrong and could be renamed to `MachineImage` as well.

Reply:
https://github.com/gardener/gardener-extension-provider-kubevirt/pull/72#discussion_r500246304